lib/grizzly/options.ex

```diff
@@ -28,7 +28,9 @@ defmodule Grizzly.Options do
           database_file: Path.t() | nil,
           indicator_handler: (Grizzly.Indicator.event() -> :ok),
           rf_region: Supervisor.rf_region() | nil,
-          power_level: {Supervisor.tx_power(), Supervisor.measured_power()} | nil
+          power_level: {Supervisor.tx_power(), Supervisor.measured_power()} | nil,
+          zwave_firmware_path: Path.t() | nil,
```

Comment:
```suggestion
          zwave_firmware: [firmware_info()],
```

Where:

```
@type firmware_info() :: %{chip_type: non_neg_integer(), path: String.t(), version: String.t()}
```

`chip_type` is `5, 7, 8` based on reading comments below, but please update.

The idea the `:zwave_firmware` list is iterated over to find a possible update. Even if we only support updating 700 series chips now, this gives some flexibility in what gets applied at runtime.